pathplannerlib/src/main/native/cpp/pathplanner/lib/path/PathPlannerPath.cpp

```diff
@@ -127,8 +129,16 @@ PathPlannerPath PathPlannerPath::fromJson(const wpi::json &json) {
 		eventMarkers.push_back(EventMarker::fromJson(markerJson));
 	}
 
+	frc::Rotation2d previewStartingRotation;
+	if (json.contains("previewStartingState")) {
+		auto jsonStartingState = json["previewStartingState"];
+		previewStartingRotation = frc::Rotation2d(
+				units::degree_t(jsonStartingState["rotation"]));
+	}
```

Comment:
Changing this to handle a null value for the starting state and use the more "safe" json api.

```suggestion
	if (!json.at("previewStartingState").is_null()) {
		auto jsonStartingState = json.at("previewStartingState");
		previewStartingRotation = frc::Rotation2d(
				units::degree_t(jsonStartingState.at("rotation").get<double>()));
	}
```

Reply:
I guess with the old way it would initialize new values if they don't exist, but I like to explicitly see what it's doing. And it matches the json usage in the rest of the library.